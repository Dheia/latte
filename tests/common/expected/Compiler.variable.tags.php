<?php
%A%
final class Template%a% extends Latte\Runtime\Template
{

	public function main(array $ʟ_args): void
	{
%A%
		echo "\n";
		$ʟ_tag[0] = '';
		echo '<';
		echo $ʟ_tmp = LR\Filters::safeTag($tag) /* line %d% */;
		$ʟ_tag[0] = '</' . $ʟ_tmp . '>' . $ʟ_tag[0];
		echo '>...';
		echo $ʟ_tag[0];
		echo '

';
		$ʟ_tag[1] = '';
		echo '<';
		echo $ʟ_tmp = LR\Filters::safeTag($tag) /* line %d% */;
		$ʟ_tag[1] = '</' . $ʟ_tmp . '>' . $ʟ_tag[1];
		echo '>...';
		echo $ʟ_tag[1];
		echo '

';
		$ʟ_tag[2] = '';
		echo '<';
		echo $ʟ_tmp = LR\Filters::safeTag('a') /* line %d% */;
		$ʟ_tag[2] = '</' . $ʟ_tmp . '>' . $ʟ_tag[2];
		echo '>';
		$ʟ_tag[3] = '';
		echo '<';
		echo $ʟ_tmp = LR\Filters::safeTag('b') /* line %d% */;
		$ʟ_tag[3] = '</' . $ʟ_tmp . '>' . $ʟ_tag[3];
		echo '>...';
		echo $ʟ_tag[3];
		echo $ʟ_tag[2];
		echo ' ';
	}
%A%
}
