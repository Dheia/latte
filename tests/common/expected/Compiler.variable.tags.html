
<foo>...</foo>

<foo>...</foo>

<a><b>...</b></a>
